cfgov/jinja2/v1/_includes/molecules/featured-content.html

```diff
@@ -1,6 +1,6 @@
 {# ==========================================================================
 
-   featured_content
+   featured_content.render()
 
```

Comment:
Shouldn't this be 'Featured Content' ? (according to how the other files have been modified)